src/document/transportation-system-plan/en/09-implementation-strategies

```diff
@@ -0,0 +1,163 @@
+---
```

Comment:
You'll need to rename this file to `09-implementation-strategies.md`.  The `.md` is important to denote this is a file for translating, otherwise this isn't showing up.